db/schema.v0.sql

```diff
@@ -312,7 +312,7 @@ CREATE TABLE IF NOT EXISTS `contact_mode` (
 -- Initialize contact modes
 -- -----------------------------------------------------
 INSERT INTO `contact_mode` (`name`)
-VALUES ('email'), ('sms'), ('call'), ('slack'), ('teams_messenger');
```

Comment:
why is teams_messenger being deleted?

Reply:
Its removed as teams_messenger is not a basic mode of contact in oncall.
Else it can be added any time in the contact_mode table to make it accessible in the UI.